terraform.tf

```
@@ -0,0 +1,21 @@
+terraform {
+
+  /* Uncomment this block to use Terraform Cloud for this tutorial
+  cloud {
+    organization = "organization-name"
+    workspaces {
+      name = "learn-terraform-locals"
+    }
+  }
+	*/
```

so it's formatted properly

```suggestion
  */
```